pdr_backend/cli/arg_tb.py

```diff
@@ -0,0 +1,94 @@
+import re
+from typing import List, Union
+
+from enforce_typing import enforce_types
+
+
+# Tick Bar
+class ArgTB:
```

Comment:
so what I'm seeing up until now is these classes are very similar but have a different prefix to the string arg itself. There are better ways to manage this without creating 3 virtually identical classes.

The easiest way is to create a class called ArgThreshold which has its string attribute and supports 3 types of prefixes (db, tb and vb). The regex check, the messages, the verification methods etc. can all use self.prefix to determine validity, instead of each class simply hardcoding it.

I don't think there is a need to define 3 classes, ArgThreshold with a `threshold_type` attribute seems enough, but of course you can also use inheritance if you need stronger type checking. In that case, I would also suggest full names for classes i.e. ArgTickThreshold instead of ArgTB etc.